### PR TITLE
Update download app text

### PR DESCRIPTION
**Pull request by imtoori**
# What this PR does

## Checklist

- [x] Unit, integration, and e2e (if applicable) tests updated
- [x] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

fixes https://github.com/grafana/oncall/issues/1523

### CODE REVIEW

File: grafana-plugin/src/containers/MobileAppConnection/parts/DownloadIcons/index.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const DownloadIcons: FC = () => (
     <Text type="primary" strong>
       Download
     </Text>
-    <Text type="primary">The Grafana IRM app is available on both the App Store and Google Play Store.</Text>
+    <Text type="primary">The Grafana OnCall app is available on both the App Store and Google Play Store.</Text>
```
Comment (joeyorlando) [inline grafana-plugin/src/containers/MobileAppConnection/parts/DownloadIcons/index.tsx:20]: general question. I see the app is published as "Grafana OnCall" in the app store. If Grafana Incident is considering adding features to the app in the future, what impact does this have on the naming of the app?

Comment (imtoori) [reply grafana-plugin/src/containers/MobileAppConnection/parts/DownloadIcons/index.tsx:20]: I think once we integrate Grafana Incindent we'll probably rename the app

Comment (Dieterbe) [reply grafana-plugin/src/containers/MobileAppConnection/parts/DownloadIcons/index.tsx:20]: that's a questions to answer during the next few months and will certainly involve our product and marketing team.  for now, it's the OnCall app.

Comment (joeyorlando) [reply grafana-plugin/src/containers/MobileAppConnection/parts/DownloadIcons/index.tsx:20]: that makes sense, thanks for the clarification guys 👍

**Review by vadimkerr — APPROVED** [review]

**Pull request merged**